src/main/java/org/broadinstitute/hellbender/tools/spark/sv/InsertSizeDistribution.java

```diff
@@ -0,0 +1,157 @@
+package org.broadinstitute.hellbender.tools.spark.sv;
+
+import com.google.api.client.repackaged.com.google.common.annotations.VisibleForTesting;
+import org.apache.commons.math3.distribution.AbstractRealDistribution;
+import org.apache.commons.math3.distribution.LogNormalDistribution;
+import org.apache.commons.math3.distribution.NormalDistribution;
+import org.apache.commons.math3.distribution.RealDistribution;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Holds the information characterizing and insert size distribution.
+ */
+public class InsertSizeDistribution implements Serializable {
+
+    private static final long serialVersionUID = -1L;
+
+    @VisibleForTesting
+    static final Type[] SUPPORTED_TYPES = { new NormalType(), new LogNormalType() };
+
+    public interface Type {
+        List<String> getNames();
+
+        AbstractRealDistribution fromMeanAndStdDeviation(final double mean, final double stddev);
+    }
+
+    public static class NormalType implements Type {
+
+        @Override
+        public List<String> getNames() {
+            return Collections.unmodifiableList(Arrays.asList("Normal", "N", "Norm", "Gauss", "Gaussian"));
+        }
+
+        @Override
+        public AbstractRealDistribution fromMeanAndStdDeviation(final double mean, final double stddev) {
+            return new NormalDistribution(mean, stddev);
+        }
+    }
+
+    public static class LogNormalType implements Type {
+
+        @Override
+        public List<String> getNames() {
+            return Collections.unmodifiableList(Arrays.asList("logN", "lnN", "logNorm", "lnNorm", "logNormal", "lnNormal"));
+        }
+
+        @Override
+        public AbstractRealDistribution fromMeanAndStdDeviation(final double mean, final double stddev) {
+            final double var = stddev * stddev;
+            final double scale = 2 * Math.log(mean) - 0.5 * Math.log(var + mean * mean); // scale = mu in wikipedia article.
+            final double shape = Math.sqrt(Math.log(1 + (var / (mean * mean)))); // shape = sigma in wikipedia article.
+            return new LogNormalDistribution(scale, shape);
+        }
+    }
+
+    private static Pattern DESCRIPTION_PATTERN =
+            Pattern.compile("^\\s*(?<name>[^\\s\\(\\)]+)\\s*\\((?<mean>[^,\\(\\)]+?)\\s*,\\s*(?<stddev>[^,\\(\\)]+?)\\s*\\)\\s*");
+
+    private final String description;
+
+    private transient AbstractRealDistribution dist;
+
+    private AbstractRealDistribution dist() {
+        initializeDistribution();
+        return dist;
+    }
+
+    public double average() {
```

Comment:
Maybe put this down with the other methods like min, max, density?  Should it be called "mean" rather than "average"?